Dockerfile

```diff
@@ -1,3 +1,23 @@
+
+###################################
+#Build stage
+FROM golang:1.9-alpine3.6 AS build-env
+
+ARG GITEA_VERSION
+ARG TAGS="bindata sqlite"
+ENV TAGS "netgo $TAGS"
```

Comment:
AFAIU [this](https://golang.org/doc/go1.5#net) this is no longer necessary and is also redundant due to `GODEBUG=netdns=go` being set in Dockerfile.